src/components/UserManagement/UserTableData.jsx

```diff
@@ -24,6 +24,14 @@ const UserTableData = React.memo(props => {
     onReset(false);
   }, [props.isActive, props.resetLoading]);
 
+  const checkPermissionsOnOwner = () => {
+    return (props.user.role === 'Owner' && !hasPermission(
+      props.role,
+      'addDeleteEditOwners',
+      props.roles,
+      props.userPermissions) ? true : false
```

Comment:
Just call "hasPermission" function and remove the ternary operators. hasPersmission() will return the correct boolean. Keep the "!" operator though.